homu/auth.py

```diff
@@ -10,19 +10,19 @@ def fetch_rust_team(repo_label, level):
     try:
         resp = requests.get(url)
         resp.raise_for_status()
-        return resp.json()["github_users"]
+        return resp.json()["github_ids"]
     except requests.exceptions.RequestException as e:
         print("error while fetching " + url + ": " + str(e))
         return []
 
 
-def verify_level(username, repo_label, repo_cfg, state, toml_keys,
+def verify_level(username, user_id, repo_label, repo_cfg, state, toml_keys,
                  rust_team_level):
     authorized = False
     if repo_cfg.get('auth_collaborators', False):
         authorized = state.get_repo().is_collaborator(username)
```

Comment:
I'd like to see the user ID used here, too. But alas, that's not supported by the [api](https://github3py.readthedocs.io/en/stable-0.9/repos.html#github3.repos.repo.Repository.is_collaborator). (It's also not supported in the [latest version](https://github3py.readthedocs.io/en/master/api-reference/repos.html#github3.repos.repo.Repository.is_collaborator).)

Reply:
It shouldn't be needed, as `is_collaborator` checks if the user is authorized in realtime.

The issue that prompted me to write this PR is that the Team API can become stale if users change their username. That's not a problem with `is_collaborator`.